modules/apps/frontend-taglib/frontend-taglib-clay/package.json

```diff
@@ -57,5 +57,5 @@
 		"checkFormat": "liferay-npm-scripts check",
 		"format": "liferay-npm-scripts fix"
 	},
-	"version": "14.1.1"
+	"version": "14.2.0"
```

Comment:
I don't think there should be no versions increased with this PR. We're only making changes on JSPs, not making any changes in taglib API or module API. Please remove version increases and double-check by running `gradle baseline` on `frontend-taglib-clay`.